packages/stylelint-plugin/src/stylelintPlugin.spec.ts

```diff
@@ -619,50 +670,30 @@ const testExamples: TestExample[] = [
     testName:
       "transitive tokens are fixed correctly when their usages are negated, and their replacements are simple",
     input:
-      '@import "~@kaizen/design-tokens/sass/spacing"; $foo: $kz-spacing-md; .foo { top: -$foo; }',
+      '@import "~@kaizen/design-tokens/sass/spacing"; $foo: $spacing-md; .foo { top: -$foo; }',
     expectedOutput:
-      '@import "~@kaizen/design-tokens/sass/spacing"; $foo: $kz-spacing-md; .foo { top: calc(-1 * #{$spacing-md}); }',
-    expectedWarnings: 1,
+      '@import "~@kaizen/design-tokens/sass/spacing"; $foo: $spacing-md; .foo { top: calc(-1 * #{$spacing-md}); }',
+    expectedWarnings: 0,
```

Comment:
Interesting that the `expectedWarnings` used to be 1, this feels like a bug we missed

Reply:
This is because we're not using a deprecated token anywhere in the new test anymore.
If the warning messages we're explicit in the prior this would have been a bit more obvious!